README.md

```diff
@@ -18,78 +66,47 @@ Use the hosted package:
 }
 ```
 
-Wrap your function handler
-```
-'use strict';
+## Step 2: Locate and set the ingest endpoint
 
-const signalFxLambda = require('signalfx-lambda');
+By default, this function wrapper will send data to the us0 realm. As a result, if you are not in us0 realm and you want to use the ingest endpoint directly, then you must explicitly set your realm. To set your realm, use a subdomain, such as ingest.us1.signalfx.com or ingest.eu0.signalfx.com.
 
-exports.handler = signalFxLambda.wrapper((event, context, callback) => {
-  ...
-});
-```
+To locate your realm:
 
-Using async/await,
-```
-'use strict';
+1. Open SignalFx and in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
 
-const signalFxLambda = require('signalfx-lambda');
 
-exports.handler = signalFxLambda.asyncWrapper(async (event, context) => {
-  ...
-});
-```
-
-#### Configuring the ingest endpoint
-
-By default, this function wrapper will send to the `us0` realm. If you are
-not in this realm you will need to set the `SIGNALFX_INGEST_ENDPOINT` environment
-variable to the correct realm ingest endpoint (https://ingest.{REALM}.signalfx.com).
-To determine what realm you are in, check your profile page in the SignalFx
-web application (click the avatar in the upper right and click My Profile).
+## Step 3: Set environment variables
 
-
-### Environment Variables
+1. Set SIGNALFX_ACCESS_TOKEN with your correct access token. Review the following example. 
 
 ```
- SIGNALFX_AUTH_TOKEN=signalfx token
+SIGNALFX_ACCESS_TOKEN=access token
 ```
 
-Optional parameters available:
-```
-SIGNALFX_SEND_TIMEOUT=milliseconds for signalfx client timeout [1000]
+2. If you use POPS, Smart Gateway, or want to ingest directly from a realm other than us0, then you must set at least one endpoint variable. (For environment variables, SignalFx defaults to the us0 realm. As a result, if you are not in the us0 realm, you may need to set your environment variables.) You can update one of the variables below. Review the following examples.  
 
-# Change the ingest endpoint URL:
-SIGNALFX_INGEST_ENDPOINT=[https://pops.signalfx.com]
+```
+    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080
+    SIGNALFX_METRICS_URL=ingest endpoint [ default: https://pops.signalfx.com ]
 ```
 
-### Metrics and dimensions sent by the wrapper
+To learn more, see: 
+  * [SignalFx Point of Presence Service (POPS)](https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.signalfx.point.of.presence.service.(pops).html)
+  * [Deploying the SignalFx Smart Gateway](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html)
+        
+    
+3. (Optional) Set additional environment variable. Review the following examples.  
 
-The Lambda wrapper sends the following metrics to SignalFx:
+```
+    SIGNALFX_SEND_TIMEOUT=timeout in seconds for sending datapoint [ default: 0.3 ]
```

Comment:
for nodejs, it is in milliseconds, as explained in the old version

Reply:
This has been updated.  Do we know the default?